src/api/exam.api.ts

```
@@ -5,11 +5,29 @@ const examApi = {
+        },
+      });
+    }   
+  },
+  edit(payload: FormData) {
+    {
+      return axiosClient.put(baseUrl, payload, {
+        headers: {
+          "Content-Type": "multipart/form-data",
+        },
+      });
+    }   
```

Biome

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)